[WPE][GTK] API test `TestWebKitWebView` `/webkit/WebKitWebView/is-web-process-responsive` is a timeout

obyknovenius opened the pull request:
#### b09b2477ece478964e92ba3323d5a19e278feaed
<pre>
[WPE][GTK] API test `TestWebKitWebView` `/webkit/WebKitWebView/is-web-process-responsive` is a timeout
<a href="https://bugs.webkit.org/show_bug.cgi?id=258810">https://bugs.webkit.org/show_bug.cgi?id=258810</a>

Reviewed by Michael Catanzaro.

In this test, we wait several times for some events to occur:

1. Wait for the first page to finish loading.
2. Wait one second for JS loop to start executing.
3. Wait until the web process becomes unresponsive.
4. Wait for the second page to finish loading.

The responsive timer alone takes at least 3 seconds to detect that page
is unresponsive.

Altogether can easily exceed the default 5 seconds threshold for
`run-wpe-test` to mark the tests as a timeout.

The same applies to
`/webkit/WebKitWebView/terminate-unresponsive-web-process` test.

Marking these tests as `slow` gives ten times more time for the tests
to finish, which should be more than enough.

* Tools/TestWebKitAPI/glib/TestExpectations.json:

Canonical link: <a href="https://commits.webkit.org/265723@main">https://commits.webkit.org/265723@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/27d168a8576ef00791574284192f2785d3daede7

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/11720 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/11919 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/12288 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/13364 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/11164 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/11739 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/14308 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/11900 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/14030 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/11884 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/12723 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/9944 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/13785 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/10015 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/10640 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/17774 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/11092 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/10795 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/13969 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/11197 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/9242 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/10374 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/2818 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/14654 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/11054 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

mcatanzaro reviewed (approved):
OK, but the title of this bug `[WPE]` implies this is not a problem for GTK, but it is. Use `[GLib]` in the title to make clear that the problem affects both ports.

(I maybe failed to catch this for your previous fixes. :)

obyknovenius committed:
[WPE][GTK] API test `TestWebKitWebView` `/webkit/WebKitWebView/is-web-process-responsive` is a timeout
https://bugs.webkit.org/show_bug.cgi?id=258810

Reviewed by Michael Catanzaro.

In this test, we wait several times for some events to occur:

1. Wait for the first page to finish loading.
2. Wait one second for JS loop to start executing.
3. Wait until the web process becomes unresponsive.
4. Wait for the second page to finish loading.

The responsive timer alone takes at least 3 seconds to detect that page
is unresponsive.

Altogether can easily exceed the default 5 seconds threshold for
`run-wpe-test` to mark the tests as a timeout.

The same applies to
`/webkit/WebKitWebView/terminate-unresponsive-web-process` test.

Marking these tests as `slow` gives ten times more time for the tests
to finish, which should be more than enough.

* Tools/TestWebKitAPI/glib/TestExpectations.json:

Canonical link: https://commits.webkit.org/265723@main
